src/components/app.jsx

```diff
@@ -65,8 +80,12 @@ const App = React.createClass({
                 </div>
                 <div className="col-md-9">
                   <p id="baseline">
-                    {Object.keys(variables).length} variables et {Object.keys(parameters).length} paramètres
-                    référencés pour modéliser le système socio-fiscal français.
+                    <FormattedMessage {...messages.header} />
+                    <br/>
```

Comment:
Multiplying `<br/>` to increase the distance between 2 texts blocks is not great.
Maybe each should be a `p` and we could set a margin in css ?